simglucose/simulation/scenario.py

```diff
@@ -40,11 +40,14 @@ def __init__(self, start_time=None, scenario=None):
         self.scenario = scenario
 
     def get_action(self, t):
-        times, actions = tuple(zip(*self.scenario))
-        times2compare = [parseTime(time, self.start_time) for time in times]
-        if t in times2compare:
-            idx = times2compare.index(t)
-            return Action(meal=actions[idx])
+        if self.scenario:
+            times, actions = tuple(zip(*self.scenario))
+            times2compare = [parseTime(time, self.start_time) for time in times]
+            if t in times2compare:
+                idx = times2compare.index(t)
+                return Action(meal=actions[idx])
+            else:
```

Comment:
This else is unnecessary since the if branch has a return.

Reply:
It actually is necessary - if I remove the else branch, the test_sim_engine.py test fails. 👀